repo/fsrepo/fsrepo_test.go

```diff
@@ -30,31 +30,7 @@ func TestInitIdempotence(t *testing.T) {
 func TestRemove(t *testing.T) {
 	t.Parallel()
 	path := testRepoPath("foo", t)
-	assert.Nil(Remove(path), t, "should be able to remove after closed")
-}
-
-func TestCannotRemoveIfOpen(t *testing.T) {
```

Comment:
wait, why did this test go away? one proc may have the repo open-- another proc must not be able to remove it.